src/legacy/components/TransactionConfirmationModal/index.tsx

```diff
@@ -70,8 +73,18 @@ export function TransactionConfirmationModal({
           currencyToAdd={currencyToAdd}
         />
       ) : (
-        content && content()
+        content?.()
       )}
     </GpModal>
   )
 }
+
+function getWidth(hash: string | undefined, activityDerivedState: ActivityDerivedState | null): number {
+  if (activityDerivedState && getActivityState(activityDerivedState) === 'filled') {
+    return 400
+  }
+  if (hash) {
+    return 850
+  }
+  return 470
```

Comment:
im curious what are these 2 cases, the 850 and 470
maybe a comment giving them a name, or a named const would help